### DISCUSSION TITLE
SEB and .mp4 video

### DISCUSSION THREAD

**Discussion by Cristobal-84** [SafeExamBrowser/seb-win-refactoring#362] [Q&A]
Hi,
I have a review page in which there is a video in .mp4.
When the video is hosted on youtube there is no problem, however it does not work when I host it on our own server.
Any idea what it could be?

**Comment by dbuechel** [comment]
SEB 3.x does not support proprietary codecs, please refer to #102.

**Reply by Cristobal-84** [reply]
Thank you very much for the answer.

**Reply by danschlet** [reply]
YouTube automatically converts uploaded videos to other formats and detects which formats to serve to a web browser (for Chromium it might use WebM). You could use a JavaScript video player ([see for example this page](https://openbase.com/categories/js/best-javascript-video-player-libraries)) which can also automatically choose which format to play (but I guess you would need to convert videos using another tool first). Note that SEB for Mac/iOS might not support WebM or only on latest macOS/iOS versions (but instead it supports MP4/H.264/AAC).

**Comment by mlohnen** [comment]
Since SEB also uses Edge Webview2 you can start MS Edge in kiosk mode as alternative for playing MP4 files in the browser:

C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe --kiosk https://www.examplesite.com

In the browser tab you set:
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/68436131/f9e969b0-69ec-484d-a4e4-7818d9ab448d)

And in the applications tab you add msedge.exe as autostart application in kiosk mode:
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/68436131/01188b75-7a56-4548-8746-504b5e03ed1e)
